README.md

```diff
@@ -1 +1,104 @@
+# webpack-sane-compiler-reporter
 
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url] [![Greenkeeper badge][greenkeeper-image]][greenkeeper-url]
+
+[npm-url]:https://npmjs.org/package/webpack-sane-compiler-reporter
+[npm-image]:http://img.shields.io/npm/v/webpack-sane-compiler-reporter.svg
+[downloads-image]:http://img.shields.io/npm/dm/webpack-sane-compiler-reporter.svg
+[travis-url]:https://travis-ci.org/moxystudio/webpack-sane-compiler-reporter
+[travis-image]:http://img.shields.io/travis/moxystudio/webpack-sane-compiler-reporter/master.svg
+[codecov-url]:https://codecov.io/gh/moxystudio/webpack-sane-compiler-reporter
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/webpack-sane-compiler-reporter/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/webpack-sane-compiler-reporter
+[david-dm-image]:https://img.shields.io/david/moxystudio/webpack-sane-compiler-reporter.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/webpack-sane-compiler-reporter?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/webpack-sane-compiler-reporter.svg
+[greenkeeper-image]:https://badges.greenkeeper.io/moxystudio/webpack-sane-compiler-reporter.svg
+[greenkeeper-url]:https://greenkeeper.io
+
+Beautiful reporting for [webpack-sane-compiler](https://github.com/moxystudio/webpack-sane-compiler) compilation events.
+
+
+## Installation
+
+`$ npm install webpack-sane-compiler-reporter --save-dev`
+
+## Usage
+
+```js
+const withReporting = require('webpack-sane-compiler-reporter');
+
+const reporter = withReporting(compiler, {/* options */});
+
+// call compiler.run() or compiler.watch() to start a compilation and start outputing reports
+```
+
+### Available options:
+
+| Name   | Description   | Type     | Default |
+| ------ | ------------- | -------- | ------- |
+| events | Which events to report | object | `{ begin: true, success: true, error: true }` |
+| messages | The message to print for each compilation event | object | `{ begin: 'Compiling...', success: 'Compilation succeeded', error: 'Compilation failed' }` |
+| stats | Display webpack stats after each compilation | boolean/string (`true`, `false` or `'once'`) | `true` |
+| statsIndentation | The number of spaces to indent the stats output by | number | `4` |
```

Comment:
Perhaps the indentation could be done in the isomorphic reporter with https://github.com/sindresorhus/indent-string